README.md

```diff
@@ -1,17 +1,16 @@
 ![Logo](https://github.com/user-attachments/assets/8c1534b8-6ad6-4c84-b855-2d16bf6b774a)
 
 # beamng.cluster
-BeamNG.cluster is a Node.js based server control panel for BeamMP servers. This project aims to take a simple server-manager idea and create a multi-server administration tool using an intuitive Dashboard. 
+BeamNG.cluster is a Node.js based server control panel for [BeamMP](https://github.com/beammp) servers. This project aims to take a simple server-manager idea and create a multi-server administration tool using an intuitive Dashboard. 
 
 # Media
-
-> Static prototype
+###### Static prototype
 
 ![image](https://github.com/user-attachments/assets/a0739b20-b64c-4ab9-887a-9ba168d429b1)
 
 ![image](https://github.com/user-attachments/assets/f1877b6c-b451-45c7-938d-772c09c67aa1)
 
-# Todo
+# TO-DO
```

Comment:
Using `Todo` or `TODO` is acceptable, I don't believe it needs to be hyphonated.